DNN Platform/Library/Entities/Portals/StyleColor.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Entities.Portals
+{
+    using System;
+    using System.Text.RegularExpressions;
+
+    using DotNetNuke.Abstractions.Portals;
+
+    /// <summary>
+    /// Represents a CSS color and its components.
+    /// </summary>
+    public struct StyleColor : IStyleColor

```

Comment:
I see that `StyleColor` is a struct: in PortalStyles all the properties use `IStyleColor`. When a struct is being converted to an interface, it's being boxed:  it's still allocating to the heap.

Reply:
Should we just move StyleColor to the abstractions project and kill IStyleColor?
Or do we want to only keep interfaces in that project ?

@bdukes any thoughts on this ?

Reply:
I'm okay with it being in the abstractions project

Reply:
Cool cool, done in https://github.com/dnnsoftware/Dnn.Platform/pull/5851/commits/f74f0e71e2df8969b614e9af4f246851ebe25877